guides/v2.2/cloud/docker/docker-config.md

```diff
@@ -223,6 +223,9 @@ The `{{site.data.var.ct}}` version 2002.0.18 and later supports developer mode.
     ```bash
     bash ./mutagen.sh
     ```
+    
+    {: .bs-callout-info}
+    If you running docker on Windows and session start was failed, change `--symlink-mode` to `portable` in the `mutagen.sh` file.
```

Comment:
```suggestion
   If you use `docker-sync` for file synchronization, skip this step.  If you host your Docker environment on Windows and the session start fails, update the `mutagen.sh` file to change the  `--symlink-mode` to `portable`.
```

Reply:
They are described in the mutagen documentation https://mutagen.io/documentation/synchronization/symbolic-links/
Maybe we can put a link into our devdocs.

Reply:
I see! Okay, don't worry about that here, then. I will consider it in the changes I am making for later. Thanks!